rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -66,24 +66,19 @@ holland_git_repo_plugins:
 
 holland_pip_wheel_name: holland
 
+holland_packages:
+  - git
+
 holland_venv_enabled: True
 holland_venv_bin: "/openstack/venvs/holland-{{ rpc_release }}/bin"
 holland_venv: "/openstack/venvs/holland-{{ rpc_release }}"
 
 holland_pip_dependencies:
   - MySQL-python
 
-hwraid_apt_repo_url: "http://mirror.rackspace.com/hwraid.le-vert.net/ubuntu/"
-
-hwraid_apt_repos:
-  - { repo: "deb {{ hwraid_apt_repo_url }} {{ ansible_lsb.codename }} main", state: "present" }
-
-hwraid_apt_keys:
-  - { url: "http://mirror.rackspace.com/hwraid.le-vert.net/ubuntu/hwraid.le-vert.net.gpg.key", state: "present" }
-
-hwraid_apt_packages:
-  - megacli
-  - lsiutil
+hwraid_apt_package_urls:
```

Comment:
Yeah, I'd prefer to see the Trusty packages used for Trusty and the Wily packages used for Xenial (in the absence of Xenial packages). I think this would be safer.